Build.cmake

```diff
@@ -23,12 +23,17 @@ SET(ACTIVE_CPP0xx_DIR "Release")
 IF ("${CMAKE_CXX_COMPILER_ID}" MATCHES ".*Clang")
    MESSAGE("")
    MESSAGE("cmake for Clang ")
-   IF (APPLE)
+   SET(CMAKE_CXX_FLAGS "-Wall -std=c++11 -stdlib=libc++ -Wunused -D_GLIBCXX_USE_NANOSLEEP")
+   IF (${CMAKE_SYSTEM} MATCHES "FreeBSD-([0-9]*)\\.(.*)")
+       IF (${CMAKE_MATCH_1} GREATER 9)
+           set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
```

Comment:
Yes, pthread is required (only for "examples" and "test performance" compilations), otherwise you will get:

>[ 39%] Linking CXX executable g3log-FATAL-choice
>/usr/local/bin/cmake -E cmake_link_script CMakeFiles/g3log-FATAL-choice.dir/link.txt --verbose=1
    /usr/bin/CC -Wall -std=c++11 -stdlib=libc++ -Wunused -D_GLIBCXX_USE_NANOSLEEP -O3 -DNDEBUG CMakeFiles/g3log-FATAL-choice.dir/example/main_fatal_choice.cpp.o -o g3log-FATAL-choice libg3logger.a -lexecinfo
    CMakeFiles/g3log-FATAL-choice.dir/example/main_fatal_choice.cpp.o: In function ZNSt3__124__make_async_assoc_stateIvNS_12__async_funcIPFvNS_8functionIFvvEEEEJPS3_EEEEENS_6futureIT_EEOT0': /ara/incoming/g3log-master/example/main_fatal_choice.cpp:(.text.ZNSt3__124__make_async_assoc_stateIvNS_12__async_funcIPFvNS_8functionIFvvEEEEJPS3_EEEEENS_6futureIT_EEOT0[ZNSt3__124__make_async_assoc_stateIvNS_12__async_funcIPFvNS_8functionIFvvEEEEJPS3_EEEEENS_6futureIT_EEOT0]+0x77): undefined reference topthread_create'
    CMakeFiles/g3log-FATAL-choice.dir/example/main_fatal_choice.cpp.o: In function ZNSt3__124__make_async_assoc_stateIvNS_12__async_funcIPFvvEJEEEEENS_6futureIT_EEOT0': /ara/incoming/g3log-master/example/main_fatal_choice.cpp:(.text.ZNSt3__124__make_async_assoc_stateIvNS_12__async_funcIPFvvEJEEEEENS_6futureIT_EEOT0[ZNSt3__124__make_async_assoc_stateIvNS_12__async_funcIPFvvEJEEEEENS_6futureIT_EEOT0]+0x6f): undefined reference topthread_create'
    libg3logger.a(logworker.cpp.o): In function kjellkod::Active::createActive()': /ara/incoming/g3log-master/src/logworker.cpp:(.text._ZN8kjellkod6Active12createActiveEv[_ZN8kjellkod6Active12createActiveEv]+0x6f): undefined reference topthread_create'
    CC: error: linker command failed with exit code 1 (use -v to see invocation)
    *** Error code 1